include/proposals/proposals_base.hpp

```diff
@@ -5,6 +5,10 @@
 namespace ProposalsCommon {
   constexpr name type_ref_setting = name("r.setting");
   constexpr name type_ref_code = name("r.code");
+  constexpr name type_prop_alliance = name("p.alliance");
```

Comment:
How about adding referendums as another proposal type here?

I think that was the idea, to combine the two